dynawo/sources/Modeler/Common/DYNDynamicData.cpp

```diff
@@ -281,9 +281,13 @@ DynamicData::associateParameters() {
 }
 
 shared_ptr<ParametersSet>
-DynamicData::getParametersSet(const string& parFile, const string& parId) {
+DynamicData::getParametersSet(const std::string& modelId, const string& parFile, const string& parId) {
```

Comment:
It may be better to have either string or std::string for all argument declarations ?

Reply:
Fixed